packages/sunburst/src/Sunburst.tsx

```diff
@@ -25,6 +25,8 @@ const InnerSunburst = <RawDatum,>({
     data,
     id = defaultProps.id,
     value = defaultProps.value,
+    innerRadius = defaultProps.innerRadius,
+    renderRootNode = defaultProps.renderRootNode,
```

Comment:
I don't think we should use the same rendering as for the arcs for the root node, it generates a shape that doesn't really make sense:

![CleanShot 2025-04-18 at 21 34 05@2x](https://github.com/user-attachments/assets/4b8ba718-b7aa-45f4-867f-6109ec8be9c4)

Which means it cannot have a color, interactivity doesn't work...

Reply:
It's a bit better, but I think using an arc doesn't work very well tbh.
When enabling/disabling the root node, we can clearly see that the shape for the root node isn't a circle but more like a donut, since it's an arc of 360 degrees.
While the donut shape isn't necessarily critical, the impact it has on the position of the root node label is IMHO, the text should be at the center.
An option (and a bit of a hack) would be to adjust `computeArcCenter` so that it handles arcs of 360 deg with an inner radius of 0 differently (and `arcLabelsRadiusOffset` should probably be ignored for this case). That's probably the easiest option.

Reply:
If we go for this approach, we should test the position of the text in the root node, and also make sure that a ring of 360 deg still behave as before.